index.js

```diff
@@ -1,22 +1,163 @@
+               (max => 
+              max.teamId === 18).length
+              if (team5 > 2){
+               return false
+              }
+    const team6 = lineup.filter
+              (max => 
+             max.teamId === 20).length
+             if (team6 > 2){
+              return false
+             } 
+    const team7 = lineup.filter
+             (max => 
+            max.teamId === 22).length
+            if (team7 > 2){
+             return false
+            }
+    const team8 = lineup.filter
+            (max => 
+           max.teamId === 27).length
+           if (team8 > 2){
+            return false
+           }
+       
+    //max players from a game   
+   const gameId1 = lineup.filter
+           (max => 
+               max.gameId === 123).length
+                if (gameId1 > 3){
+                    return false
+                }
+   const gameId2 = lineup.filter
+                (max => 
+               max.gameId === 115).length
+               if (gameId2 > 3){
+                return false
+               }           
+   const gameId3 = lineup.filter
+               (max => 
+              max.gameId === 101).length
+              if (gameId3 > 3){
+               return false
+              } 
+   const gameId4 = lineup.filter
+              (max => 
+             max.gameId === 134).length
+             if (gameId4 > 3 ){
+              return false
+             }
+   const gameId5 = lineup.filter
+             (max => 
+            max.gameId === 131).length
+            if (gameId5 > 3){
+             return false
+            } 
+   const gameId6 = lineup.filter
+            (max => 
+           max.gameId === 119).length
+           if (gameId6 > 3){
+            return false
+           }
+   const gameId7 = lineup.filter
+           (max => 
+          max.gameId === 126).length
+          if (gameId7 > 3){
+           return false
+          } 
+    const gameId8 = lineup.filter
+          (max => 
+         max.gameId === 123).length
+         if (gameId8 > 3){
+          return false
+         } 
+    const gameId9 = lineup.filter
+         (max => 
+        max.gameId === 124).length
+        if (gameId9 > 3){
+         return false
+        } 
+// players form each position
+    const outfield = lineup.filter(
+                     pos => 
+                     pos.position === 'OF').length
+                     if (outfield != 3){
+                         return false
+                     }
+                       const pitcher = lineup.filter(
+                        pos => 
+                        pos.position === 'P').length
+                        if (pitcher != 1){
+                            return false
+                        }
+                       const catcher = lineup.filter(
+                            pos => 
+                            pos.position === 'C').length
+                            if (catcher != 1){
+                                return false
```

Comment:
Be sure to use your eslint to clean up your code....next time we meet lets look and make sure that you actually have that extension in your VSCode

Reply:
to be honest I totally forgot about using enlint, thanks for the reminder